pvactools/lib/run_argument_parser.py

```diff
@@ -111,6 +111,12 @@ def __init__(self, tool_name, input_file_help):
             default=1,
             help="Number of threads to use for parallelizing peptide-MHC binding prediction calls.",
         )
+        parser.add_argument(
+            "--netmhc_pan_version",
+            choices=["4.3", "4.2", "4.1", "4.0"],
+            default=None,
```

Comment:
If you set the default here to "4.1" you can remove some of the logic that checks whether the netmhcpan version is set. 4.1 is the version that is already used.